backend/null.pm

```diff
@@ -0,0 +1,42 @@
+# Copyright © 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+package backend::null;
```

Comment:
Maybe add a comment that this backend is for internal testing?

Reply:
Yeah I think we don't have a consistent list of backends extracted into documentation. But actually I would not say that the backend is meant to be used "only for internal testing". It can very well be used for "external testing", e.g. run tests of os-autoinst-distri-opensuse against that.